### ISSUE TITLE
Issue with $variables and golang

### ISSUE THREAD

**Issue by whatisgravity** [Depado/govue#3]
How did you resolve the issue with golang templates trying to read the {{ $index }} variable in the template and failing to find the value? The variables inside the vue v-(thing) tags in the html but all of the {{ }} variables are picked up by go and just fail. Any help is appreciated.

**Comment by depado** [comment]
Hi. I guess I just didn't put any of those in the html template file. All these variables are declared and used in the `app.js` file, and thus are not interpreted by the Golang template engine. I think that's what you're looking for. 

Also, you can [have a look there](http://stackoverflow.com/questions/20459706/golang-and-angularjs-template-conflict) which points to [html.template.Template.Delims](https://golang.org/pkg/html/template/#Template.Delims).

Also the whole idea of vue is to do the rendering on the client side (quite the Angular approach, see the SO question above), so I guess that you're trying to mix both server rendering and client rendering which is (in my opinion) quite a bad idea. 

If those lines don't answer your question, then I didn't understand it, and I'm sorry about that.

**Comment by whatisgravity** [comment]
Thanks, I will give that a try.

**Comment by depado** [comment]
You're welcome. Please don't use this repo as a best practice example. I quite had no idea what I was doing right there :D

**Comment by whatisgravity** [comment]
Trying just to understand and get more familiar with vue, which I have no experience with it. And just trying to familiarize myself with gin as well. Not building anything production, more just exploring different ways to construct web ui's in golang.

I got it working once I stopped loading using gin's loadHTMLGlob loading of the templates which I now understand is executing the variables as golang. I did not realize it was executing then, I expected it to compile the templates on GET/POST calls. 

Thanks.
